app-interface/manifests/rhose-fleet-manager.yml

```diff
@@ -251,6 +251,11 @@ objects:
                     secretKeyRef:
                       name: ${FLEET_MANAGER_SECRET_NAME}
                       key: SHARD_ID
+                - name: SHARD_ROUTER_CANONICAL_HOSTNAME
```

Comment:
@msmagnanijr For the `${FLEET_MANAGER_SECRET_NAME}` We seem to be simply re-binding the key to environment values. Is it simpler to simply have the following:

```
envFrom:
- secretRef:
                name: ${FLEET_MANAGER_SECRET_NAME}
```

As part of the `PodSpec` here? That would save you having to raise a PR each time we introduce a new value into the Vault.

Reply:
@robpblake  It makes sense, I will refactor this soon. tks :)